plugins/auth-backend/src/providers/providers.ts

```diff
@@ -75,4 +76,5 @@ export const defaultAuthProviderFactories: {
   awsalb: awsAlb.create(),
   bitbucket: bitbucket.create(),
   atlassian: atlassian.create(),
+  cfaccess: cfAccess.create(),
```

Comment:
Let's only add this to the above `providers` object instead, as there's no sensible default implementation due to the sign-in resolver being required.